sdk/storage/Azure.Storage.Blobs/tests/SessionRecords/BlobBaseClientTests/OpenReadAsync_Seek_Position(10,Current).json

```diff
@@ -55,33 +58,35 @@
       "ResponseHeaders": {
         "Content-Length": "0",
         "Content-MD5": "dvvPc9IVViagoIJiWkrb\u002BQ==",
-        "Date": "Wed, 07 Oct 2020 19:40:06 GMT",
-        "ETag": "\u00220x8D86AF8CB190ADE\u0022",
-        "Last-Modified": "Wed, 07 Oct 2020 19:40:06 GMT",
+        "Date": "Wed, 17 Feb 2021 18:45:10 GMT",
+        "ETag": "\u00220x8D8D374279DC806\u0022",
+        "Last-Modified": "Wed, 17 Feb 2021 18:45:10 GMT",
         "Server": [
           "Windows-Azure-Blob/1.0",
           "Microsoft-HTTPAPI/2.0"
         ],
         "x-ms-client-request-id": "53cd983d-0cb2-971f-8b73-71d6f0005250",
         "x-ms-content-crc64": "A33U2s4FRsw=",
-        "x-ms-request-id": "19fd17fa-501e-0064-31e1-9c2b5a000000",
+        "x-ms-request-id": "84fe38c3-101e-005a-0f5d-05bc25000000",
         "x-ms-request-server-encrypted": "true",
-        "x-ms-version": "2020-06-12"
+        "x-ms-version": "2020-06-12",
+        "x-ms-version-id": "2021-02-17T18:45:10.7244038Z"
       },
       "ResponseBody": []
     },
     {
       "RequestUri": "https://seanmcccanary3.blob.core.windows.net/test-container-52bce973-743d-f688-387a-f42ef0455460/test-blob-436b4032-bbb1-b1a9-b087-a6dcfe9afef2",
       "RequestMethod": "HEAD",
       "RequestHeaders": {
+        "Accept": "application/xml",
```

Comment:
sending accept header with HEAD request doesn't make much sense. The response won't have body.

Reply:
Discussed offline.